### DISCUSSION TITLE
How to update Uptimekuma?

### DISCUSSION THREAD

**Discussion by sampl3x** [tteck/Proxmox#2016] [General]
I want to update uptimekuma, the description says type update in the lxc container but this command is not recognized.

How can i update uptimekuma?

**Comment by X-Anonymous-Y** [comment]
> I want to update uptimekuma, the description says type update in the lxc container but this command is not recognized.
> 
> How can i update uptimekuma?

Run the installer script inside you uptimekuma lxc terminal/bash

`BASH
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/uptimekuma.sh)"
`

**Reply by gigi2006** [reply]
quick question. are all installation scripts also the update scripts?

**Reply by X-Anonymous-Y** [reply]
I thnik not all. Best way is to test it :)

Uptimekuma, adguard, nginxproxymanager, alpine, debian, ubuntu works ;)
